src/Temporalio/Converters/DefaultPayloadConverter.cs

```diff
@@ -87,7 +79,6 @@ protected DefaultPayloadConverter(params IEncodingConverter[] encodingConverters
 
         // KeyedCollection not worth it and OrderedDictionary not generic. So we'll expose the
         // collection and maintain an internal dictionary.
-
```

Comment:
Looks like this line removal broke lint check, may need to add it back. Also note since I merged `main` into your branch, you'll want to `pull` before making this change.

Reply:
@cretz mobile edits ftw :) 

Should be good for you to retrigger CI